libs/payments/customer/src/lib/invoice.manager.ts

```diff
@@ -142,6 +144,18 @@ export class InvoiceManager {
       );
     }
 
+    // TODO: Replace countryCode value with LocationPicker value
```

Comment:
Unsure of the intent of this comment, but customer_shipping.address.country is set by the tax logic. I believe this comment should be removed